sklearn/impute/tests/test_impute.py

```diff
@@ -1008,6 +1008,13 @@ def test_missing_indicator_new(missing_values, arr_type, dtype, param_features,
     X_fit_expected = X_fit_expected.astype(dtype)
     X_trans_expected = X_trans_expected.astype(dtype)
 
+    indicator_test_ft = MissingIndicator(missing_values=missing_values,
+                                         features=param_features,
+                                         sparse=False)
+    fit_transformer = indicator_test_ft.fit(X_fit).transform(X_fit)
+    fit_transformer_ = indicator_test_ft.fit_transform(X_fit)
+    assert_array_equal(fit_transformer, fit_transformer_)
```

Comment:
I'm pretty sure this is already tested. If this change is improving efficiency, and not changing functionality, it does not require a new test.

Reply:
Ok, I'll remove it, I added it because I was asked to. And I think I don't see where it is being checked. Please correct me if I'm wrong.

Thanks.

Reply:
I think this is meant to be checked within check_estimator

Reply:
Ok, so I'll remove it.

Thanks.